src/commands.cc

```diff
@@ -1045,6 +1046,37 @@ Vector<String> params_to_shell(const ParametersParser& parser)
     return vars;
 }
 
+String deindent_docstring(StringView docstring)
```

Comment:
I'd rather have this in string_utils.cc, and not referecing docstrings explicitely. Maybe called 'trim_indent' ?

Reply:
I don't think "trim" is specific enough, and this function is only used on docstrings hence the "docstring" in the name.